cgs_vmc/training.py

```diff
@@ -167,7 +167,7 @@ def build_opt_ops(
     psi_target = target_wavefunction(configs)
 
     loss = tf.reduce_mean(
-        tf.squared_difference(psi, psi_target * np.sqrt(2**n_sites)) /
+        tf.squared_difference(psi, psi_target * hparams.scale) /
```

Comment:
I haven't thought about it before, but it makes more sense to put scale up to the `psi_target`, simply improves readability. (otherwise it's confusing that we are matching scaled target)